parrot/src/pfs_table.cc

```diff
@@ -660,10 +677,26 @@ pfs_file * pfs_table::open_object( const char *lname, int flags, mode_t mode, in
 				file = pfs_cache_open(&pname,flags,mode);
```

Comment:
I believe we need to check for `EISDIR` here as well as the other `pfs_cache_open`. At this point, I think it's best to have an `open_directory` function that handles the `flags` logic...